installer/update-inno-setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # Download the most recent Inno Setup version.
-installer="is-unicode.exe"
+installer="is.exe"
```

Comment:
There's no longer an `is-unicode.exe` file. [Details](https://jrsoftware.org/isdl.php#stable):

> Change in default behavior: Starting with Inno Setup 6 there's only one version available: Unicode Inno Setup. Unicode Inno Setup has been available for 9 years but in case you have not yet updated to it: please see the Unicode Inno Setup topic in the help file for more information. Basically, unless you're using [Code] to make DLL calls with string parameters you shouldn't have to make any changes to your script.

Reply:
I did the same in dscho/build-extra@commit 114305fcd781c6ea3149436896faa53307dab92c, and I also added `//CURRENTUSER`, no idea why, I forgot the details (probably because it's been over a year that I dabbled with the InnoSetup upgrade).